hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/transport/server/GrpcXceiverService.java

```diff
@@ -107,9 +108,20 @@ public StreamObserver<ContainerCommandRequestProto> send(
 
       @Override
       public void onNext(ContainerCommandRequestProto request) {
+        final DispatcherContext context;
```

Comment:
I think we got to do the same in ContainerStateMachine.readStateMachineData. Otherwise there'll be memory leak. Not sure if I should put this comment in https://github.com/apache/ozone/pull/5805